lib/Twig/Extensions/Extension/Intl.php

```diff
@@ -55,12 +55,17 @@ function twig_localized_date_filter(Twig_Environment $env, $date, $dateFormat =
         'full' => IntlDateFormatter::FULL,
     );
 
+    $calendarValues = array(
+        'gregorian' => IntlDateFormatter::GREGORIAN,
+        'traditional' => IntlDateFormatter::TRADITIONAL,
```

Comment:
No, ICU project only provide two calendars, Gregorian  and non-gregorian calendars(traditional).
If some one needs Persian(Jalali) calendar she/he should add '@calendar=persian' at the end of locale string. for example: 'fr_FR@calendar=persian'